app/__init__.py

```diff
@@ -151,6 +152,10 @@ def override_url_for():
     application.config['THEME_PATHS'] = os.path.dirname(os.path.abspath(__file__))
     Themes(application, app_identifier="surveyrunner")
 
+    @application.teardown_appcontext
+    def shutdown_session(exception=None):
```

Comment:
It should appear on next build again. I copied what was on pylint which you'd assume is correct?